site/assets/scss/_clipboard-js.scss

```diff
@@ -25,6 +27,7 @@
   padding: .25rem .5rem;
   @include font-size(.65em);
   color: $primary;
+  white-space: nowrap;
```

Comment:
This seems unneeded

Reply:
It is needed, or the Try it text is rendered as

⚡️
Try
It

Weird, no idea why. This line fixes it

Reply:
I'm pretty sure we can use one of our utilities, though, can't we?

Reply:
Which utility? 😅

Reply:
https://getbootstrap.com/docs/5.1/components/buttons/#disable-text-wrapping

Reply:
> In Sass, you can set $btn-white-space: nowrap to disable text wrapping for each button.

Isn't `white-space: nowrap` shorter and clearer? 😅

Is this change really needed?

Reply:
Use `.text-nowrap`, as mentioned in the first sentence—instead of adding CSS there :)